### DISCUSSION TITLE
Not able  to configure Email  apche superset 2.0.0

### DISCUSSION THREAD

**Discussion by Nixan04** [apache/superset#21926] [Q&A / Help]
When i was trying to configure Email in superset and not able to find configure option  "ENABLE_SCHEDULED_EMAIL_REPORTS = True" why it is not avaliable ?
superset version 2.0.0

(https://user-images.githubusercontent.com/85859071/197684203-53b03dc5-311f-4df8-93b6-4561ab994d23.png)

**Comment by sfirke** [comment]
You are in the part of the docs where it says "version 0.38 and below" and "Old Reports feature."  Try following the steps in the half of the document preceding that.

**Reply by Nixan04** [reply]
Even after the configuration "ENABLE_SCHEDULED_EMAIL_REPORTS = True" not able view these in my dashboard
Manage > Dashboard Emails
Manage > Chart Email Schedules
![2](https://user-images.githubusercontent.com/85859071/198025136-81711d90-0d4a-4b26-ae7c-70f2e222cb4a.png)
![3](https://user-images.githubusercontent.com/85859071/198025186-1fd3a0b2-74e8-430f-98a2-81a4f088caeb.png)

**Comment by sfirke** [comment]
`ENABLE_SCHEDULED_EMAIL_REPORTS` flag is obsolete, for version 1.0.0 and greater you want `ALERT_REPORTS` per https://superset.apache.org/docs/installation/alerts-reports/#requirements.  It looks like you have configured that because I see Alerts & Reports in that screenshot.  If you click that, can you add an alert with +ALERT button on that screen?  Or you can do it from a dashboard like this 
![image](https://user-images.githubusercontent.com/7569808/198059320-cfd938d0-118d-4bb6-ae1b-f5823762f977.png)

**Reply by Nixan04** [reply]
But still I am not getting  Dashboard Emails and  Chart Email Schedules in my setting why is that so?? 
I have set true for both ENABLE_SCHEDULED_EMAIL_REPORTS and ALERT_REPORTS still same result
![4](https://user-images.githubusercontent.com/85859071/198198486-849e8865-1a77-48b6-9723-714d8ec20bd8.png)

**Reply by sfirke** [reply]
*Dashboard Emails* and *Chart Email Schedules* are obsolete menu options.  You have *Alerts & Reports* on your menu which is the correct option in version 2.0.0.  It seems you are still reading the part of that docs page where it says "Old Reports feature".  Ignore everything below that line.

**Comment by Nixan04** [comment]
oaky thanks...
But even after setting "Alert and Report" i am not getting any Email. I  have done all the mail configure even thought . Not able to view logs of "Alert and Report"
![1](https://user-images.githubusercontent.com/85859071/198503585-39b80e74-d679-4319-901e-8c268122f688.png)
![5](https://user-images.githubusercontent.com/85859071/198503855-d41022f5-b771-4e58-ad30-c278b49b0115.png)

**Reply by sfirke** [reply]
We have solved the issue of which flag to enable Alerts and Reports so I think this discussion is answered.  Troubleshooting _why_ your reports are not working is a complicated process with lots of possibilities, depends on how you have deployed Superset, etc.  I suggest posting that question in the Superset Slack.
